app/controllers/db/rollbacks_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Db
+  class RollbacksController < ApplicationController
+
+    expose(:rollbacks) do
+      Db::RollbackDecorator.decorate_collection(
+        Db::Rollback.reverse_order(:clear_date)
+                        .page(params[:page])
+      )
+    end
+
+    def create
+      record = ::Db::Rollback.new(
+        clear_date: clear_date,
+        issue_date: Time.zone.now,
+        state: "P"
+      )
+
+      if record.save
+        ::Db::RollbackWorker.perform_async(record.id) unless Rails.env.test?
+
+        redirect_to db_rollbacks_path,
+                    notice: "Rollback was successfully scheduled for #{record.clear_date.strftime("%e %b %Y")}. Please wait!"
```

Comment:
Can we use the formatted string type we added :uk

Reply:
@matthewford good catch - sure, will update